tldw_Server_API/app/core/config_sections/stt.py

```
@@ -0,0 +1,212 @@
+from __future__ import annotations
+
+import json
+import os
+from dataclasses import dataclass
+from typing import Mapping
```

Ruff (0.15.9)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)